Documentation/Administration/Docker/DockerComposeDemo/_codesnippets/_docker-compose.yaml

```diff
@@ -0,0 +1,28 @@
+version: '3.9'
```

Comment:
Defining a version is deprecated since the introduction of `docker compose`

Reply:
removed it